File: Vocal.MobileApp/src/models/response/InitResponse.ts
import {SettingsResponse} from './SettingsResponse';
import {UserResponse} from './UserResponse';
import {TalkResponse} from './TalkResponse';
import {KeyValueResponse} from './KeyValueResponse';

export class InitResponse{
    Settings: SettingsResponse;
    Friends: Array<UserResponse>;
    FriendsAddedMe: Array<UserResponse>;
    Talks: Array<TalkResponse>;
    Errors: Array<KeyValueResponse<string, string>>;
}

File: backend/Vocal.Standard.Model/Response/InitResponse.cs
using System.Collections.Generic;

namespace Vocal.Model.Response
{
    public class InitResponse
    {
        public SettingsResponse Settings { get; set; }
        public List<UserResponse> Friends { get; set; }
        public List<TalkResponse> Talks { get; set; }
        public List<KeyValueResponse<string, string>> Errors { get; set; } = new List<KeyValueResponse<string, string>>();
    }
}
